app/assets/javascripts/admin/addon/routes/admin-plugins-show-schema.js

```diff
@@ -0,0 +1,35 @@
+import Route from "@ember/routing/route";
+import SiteSetting from "admin/models/site-setting";
+
+export default class AdminPluginsShowSchemaRoute extends Route {
+  async model(params) {
+    const plugin = this.modelFor("adminPlugins.show");
+
+    const [pluginSettings] = await SiteSetting.findAll({
+      plugin: plugin.id,
+    });
+
+    const setting = pluginSettings.siteSettings.find(
+      (s) => s.setting === params.setting_name
+    );
```

Comment:
Minor one but this seems quite inefficient as we should just be able to fetch a single site setting from the server.